packages/eslint-plugin/src/util/astUtils.ts

```diff
@@ -239,6 +239,77 @@ function isMemberOrOptionalMemberExpression(
   );
 }
 
+const KEYWORDS = [
```

Comment:
I'm not really sure of the value of this, as having it here means we have to remember to update items if the list ever changes, and that we're now shipping code that is already available elsewhere (since we can just import it from eslint).

Reply:
The list is also hardcoded in the original `keyword-spacing.js`
https://github.com/eslint/eslint/blob/master/lib/rules/keyword-spacing.js
Since the list can only be imported partially, we still have to hardcode some of it, and we may have to change it if a new keyword is introduced.
What do you think?

For now I will import what I can from the list eslint exports.

Reply:
Ah yes I see what you mean - that is a pain.

I think importing + concating whats missing is the best way to go, since it's a free deduplication that we'll not have to think much about after it's committed so might as well take it.

I don't think we need to worry about the deduplication check they've got in the rule however, at least for now.

Reply:
About this deduplication thingy... It was there in the original file... Maybe for a  reason. It is documented nowhere why a given keyword made it to the offical list, and another doesn't.
This leads me to think that we anticipate the possibility that the `eslint` guys may tamper with that list. In this case, I would favor crashing early, even if it adds a bit of complexity.